src/I_Ecobee1.xml

```diff
@@ -954,7 +954,7 @@ decompress_lzo_file() {
       if isEmsThermostat(lul_device) then
         return away and setOccupied(session, selection, lul_device, false) or resumeProgram(session, selection, 3)
       else
-        return setClimateHold(session, selection, lul_device, away and "away" or "home")
+        return setClimateHold(session, selection, lul_device, away and "away" or "home" or "sleep" or "smart1")
```

Comment:
This is embarrassing. I obviously did not understand what this line meant. I was trying to find where in the code the occupancy was determined and flipped the binary switch. I think I found it elsewhere so as to make the "sleep" setting also an occupied state. I guess this should be reverted